phd-tests/framework/src/artifacts/download.rs

```diff
@@ -0,0 +1,66 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+use anyhow::Context;
+use std::time::Duration;
+
+#[derive(Debug)]
+pub(super) struct DownloadConfig {
+    pub(super) timeout: Duration,
+    pub(super) buildomat_backoff: backoff::ExponentialBackoff,
+    pub(super) remote_server_uris: Vec<String>,
+}
```

Comment:
This looks like it has a duplicate in the parent module--is that by design?

Reply:
Whoops, my bad, I was going back and forth on whether or not this belonged in a module and I meant to delete the file. Good catch!